fw/http.h

```diff
@@ -394,7 +401,11 @@ typedef struct {
 	TfwConn		*conn;						\
 	void (*destructor)(void *msg);					\
 	TfwStr		crlf;						\
-	TfwStr		body;
+	TfwStr		body;						\
+	char		*body_start_data;				\
+	struct sk_buff	*body_start_skb;				\
+	TfwStr		cut;
```

Comment:
According to the patch the fields are used for responses only, so we can also make memory footprint bit smaller by placing them into `TfwHttpResp` and do not spend extra memory for requests

Reply:
Fixed